string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,25 @@
+/* exported isPalindromic */
+
+/* - make storage to store all non-alphanumeric characters
+   - make storage to store lowercased word removes all non-alphanumeric
+     characters
+   - make storage to store how long string is
+   - for each half of the word
+        - go through each letter and make sure the corresponding
+          letter is the same as the corresponding letter on the other side
+                - if one letter does not match the other
+                      - give back false
+                - if every letter matches, give back true  */
+
+function isPalindromic(string) {
+  var regex = /[^A-Za-z0-9]/g;
```

Comment:
Same deal here, you can keep this if you can explain exactly how its working to me.